lib/Conversion/TorchOnnxToTorch/DefaultDomainQtoZ.cpp

```diff
@@ -26,4 +26,20 @@ using namespace mlir::torch::onnx_c;
 // results in a lot of ONNX test cases that all reduce to the exact same
 // thing here, so we simplify.
 void mlir::torch::onnx_c::populateDefaultDomainQtoZ(
-    OnnxCustomOpConversionPattern &patterns) {}
+    OnnxCustomOpConversionPattern &patterns) {
+
+  patterns.onOp(
+      "Tan", 7, [](OpBinder binder, ConversionPatternRewriter &rewriter) {
+        Torch::ValueTensorType resultType;
+        Value operand;
+        if (binder.tensorOperand(operand) ||
+            binder.tensorResultType(resultType))
+          return failure();
+        auto loc = binder.op->getLoc();
+        Value sin = rewriter.create<Torch::AtenSinOp>(loc, resultType, operand);
+        Value cos = rewriter.create<Torch::AtenCosOp>(loc, resultType, operand);
+        rewriter.replaceOpWithNewOp<Torch::AtenDivTensorOp>(
```

Comment:
You can directly create `AtenTanOp`.

Reply:
`AtenTanOp` had not been added and I was not certain whether it was an `aten` operation in `torch`. Found a reference and added `AtenTanOp` with the corresponding lowering to `math.tan`